cc_plugin_glider/glider_dac.py

```diff
@@ -410,6 +417,109 @@ def check_dimensions(self, dataset):
+            var = self.dataset.variables[var_name]
+
+        if expected_dtype is not None:
+            if not compare_dtype(var.dtype, expected_dtype):
+                messages.append("Variable {} is expected to have a dtype of "
+                                "{}, has a dtype of {} instead"
+                                "".format(var_name, var.dtype,
+                                           expected_dtype))
+                is_passing = False
+
+
+        sub_dict = expected_format.get(var_name, {})
+        var_attrs = set(var.ncattrs())
+
+        for attr in check_attrs:
+            if attr not in var_attrs:
+                messages.append("variable {} must have a {} attribute"
+                                "".format(var_name, attr))
+                is_passing = False
+                continue
+
+            if attr == 'standard_name' and 'standard_name' in sub_dict:
+                var_std_name = var.standard_name
+                if var_std_name != sub_dict['standard_name']:
+                    messages.append("Variable {} must have a standard_name of {}".format(var_name,
+                                                                                            sub_dict['standard_name']))
+                    is_passing = False
+            elif attr == 'units':
+                raw_units = var.units
+                if 'units' in sub_dict:
+                    cur_unit = Unit(raw_units)
+                    comp_unit = Unit(sub_dict['units'])
+                    if not cur_unit.is_convertible(comp_unit):
+                        messages.append("Variable {} units attribute must be "
+                                        "convertible to {}".format(var_name,
+                                                                    comp_unit))
+                        is_passing = False
+            elif attr == '_FillValue':
+                if not compare_dtype(var.dtype, var._FillValue.dtype):
+                    messages.append("Variable {} _FillValue dtype does not "
+                                    "match variable dtype"
+                                    "".format(var_name, var._FillValue.dtype,
+                                              var.dtype))
+                    is_passing = False
+
+
+        return (is_passing, messages)
+
+
+    def check_if_monotonically_increasing(self, ds):
+       """Check if all times are monotonically increasing"""
+       # shouldn't this already be handled by CF trajectory featureType?
+       test_ctx = TestCtx(BaseCheck.MEDIUM, 'Monotonically increasing time')
+       test_ctx.assert_true(np.all(np.diff(ds.variables['time']) > 0),
+                            'Time is not monotonically increasing')
+       return test_ctx.to_result()
+
+    def check_dim_nodata(self, dataset):
+        """
+        Gets the transposed cartesian product of the depth and time
+        variables.
+        """
+
+        test_ctx = TestCtx(BaseCheck.MEDIUM, 'Non NaN count >= 2')
```

Comment:
Can you update this check name to something more descriptive like "Valid depth and time arrays"?

Also I think this is a requirement for valid DAC data so it should be a high priority check.